packages/jsii-python-runtime/src/jsii/_kernel/providers/process.py

```diff
@@ -289,13 +303,15 @@ def send(
         # Send our data, ensure that it is framed with a trailing \n
         self._process.stdin.write(b"%b\n" % (data,))
         self._process.stdin.flush()
-
+        
```

Comment:
Probably want to strip this whitespace, a well as the other bit of whitespace added.